tests/appsec/iast/sink/test_header_injection.py

```diff
@@ -2,8 +2,48 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2021 Datadog, Inc.
 
-from utils import context, features, missing_feature
-from ..utils import BaseSinkTest
+from utils import context, features, missing_feature, weblog
+from ..utils import BaseSinkTest, assert_iast_vulnerability
+
+
+class BaseTestHeaderInjectionReflectedExclusion:
```

Comment:
Could you extract the Base class to a file utils.py?
may be tests/appsec/iast/utils.py or

Reply:
Base class was initially place in `utils.py`. But, since it is only used in `test_header_injection.py` and it is closely related to these test, I decided, following a [PR comment](https://github.com/DataDog/system-tests/pull/3408#discussion_r1834610236) to place inside the test file.

What's the rationale to move it to `utils.py`?

Reply:
The CI job computation. To know what should we run when you commit something

Reply:
it's fine to have the base class inside the test file, as long as this base class is not imported in another test file. If it's imported in another test file, it **must** be placed in a `utils.py` at the same location for the reason Roberto gave.

Though, as it must not be imported, it may be a good idea to follow python convention for "private" members : 

```suggestion
class _BaseTestHeaderInjectionReflectedExclusion:
```

(will also need a rename of other occurences)

Reply:
I've applied your suggestion @cbeauchesne